import_to_es.py

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env python
+"""
+Bulk load torents from mysql into elasticsearch `nyaav2` index,
+which is assumed to already exist.
```

Comment:
This doesn't really matter. Unless you have a custom mapping, ES will create a new index on the fly when you `PUT` or `POST` to it.